### ISSUE TITLE
Fix incorrect BEM classes

### ISSUE THREAD

**Issue by warpech** [StarcounterApps/People#73]
Some classes in this project violate [BEM rules](https://docs.starcounter.io/guides/mapping-and-blending/avoiding-css-conflicts/), especially rule 7.

```
.people-address__dropdown,
.people-custom-field__dropdown,
.people-email-address__dropdown,
.people-phone-number__dropdown

/* ... */

.people-confirm__dialog,
.people-add__dialog {
```

The above classes are Elements of unexisting Blocks. This should be `people-address-dropdown`, etc.

```
.collapse-content
```

The above class is not BEMified at all.

```
.people-group-member-autocomplete,
.people-autocomplete
```

The above could be: `.people-autocomplete, .people-autocomplete--group-member`

cc @Joltrix, @threezool

**Comment by threezool** [comment]
Think i got most of them, fix in #74

**Comment by threezool** [comment]
Fixed in #74
